blobstore-fs/Makefile

```diff
@@ -0,0 +1,27 @@
+# blobstore-fs Makefile
+
+CAPABILITY_ID = "wasmcloud:blobstore_fs"
```

Comment:
All capability providers implementing the blobstore contract must use the same contract ID, `wasmcloud:blobstore` .